src/controller/java/AndroidDeviceControllerWrapper.cpp

```diff
@@ -0,0 +1,249 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+#include "AndroidDeviceControllerWrapper.h"
+
+#include <memory>
+
+using chip::DeviceController::ChipDeviceController;
+
+namespace {
+
+bool FindMethod(JNIEnv * env, jobject object, const char * methodName, const char * methodSignature, jmethodID * methodId)
```

Comment:
These functions could probably be extracted to some JNIUtil.h so that they can be reused in CHIPDeviceController-JNI.cpp. Though that change can be done later.

Reply:
Agree. Maybe a future cleanup PR.